LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 Stellar Development Foundation
+   Copyright 2024 Stellar Development Foundation
```

Comment:
Should we keep the starting year like `Copyright 2022-2024 ...`?

Reply:
It's my understanding that the docs repos have been moving around between forks/clones since the network launched. Might the SDF claims start back in 2014, and community contributions sometime thereafter? 🤔 When were the docs open-sourced / first external commit?